recipes-devtools/python/python-bson_0.4.1.bb

```diff
@@ -1,12 +0,0 @@
-DESCRIPTION = "Independent BSON codec for Python that doesn't depend on MongoDB"
```

Comment:
Commit is subsumed by other changes.

This is a valuable commit; this issue is however already addressed in
bmwcarit#509.